core/src/test/java/io/onetable/GenericTable.java

```diff
@@ -91,16 +89,16 @@ static GenericTable getInstanceWithAdditionalColumns(
       Path tempDir,
       SparkSession sparkSession,
       JavaSparkContext jsc,
-      TableFormat sourceFormat,
+      String sourceFormat,
       boolean isPartitioned) {
     switch (sourceFormat) {
-      case HUDI:
+      case "HUDI":
```

Comment:
Same question about reusing the constants in this file

Reply:
done.